AUTHORS

```diff
@@ -575,3 +575,4 @@ a license to everyone to use it as detailed in LICENSE.)
 * José Cadete <crudelios@gmail.com>
 * Cynthia K. Rey <cynthia@cynthia.dev>
 * Andrew Shaitorov <andrew.shaitorov@gmail.com>
+* on-keyday <rokenboo47803@gmail.com>
```

Comment:
It looks like you have conflict here.  If you are going to upload anther patch can you make sure the line ends in a newline?